exercises/concept-wip/crystal-hunter/spec/crystal_hunter_spec.cr

```diff
@@ -0,0 +1,64 @@
+require "spec"
+require "../src/*"
+
+describe Rules do
+  describe "bonus_points?" do
+    it "Chracter gets bonus points" do
+      Rules.new.bonus_points?(true, true).should be_true
+    end
+
+    it "Chracter gets no bonus points because no power-up is active" do
+      Rules.new.bonus_points?(false, true).should be_false
+    end
+
+    it "Chracter gets no bonus points because not touching bandit" do
+      Rules.new.bonus_points?(true, false).should be_false
+    end
+
+    it "Chracter gets no bonus points because no power-up is active and character is not touching bandit" do
```

Comment:
```suggestion
    it "Character gets bonus points" do
      Rules.new.bonus_points?(true, true).should be_true
    end

    it "Character gets no bonus points because no power-up is active" do
      Rules.new.bonus_points?(false, true).should be_false
    end

    it "Character gets no bonus points because not touching bandit" do
      Rules.new.bonus_points?(true, false).should be_false
    end

    it "Character gets no bonus points because no power-up is active and character is not touching bandit" do
```